specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -649,6 +649,16 @@
             "name"
           ]
         },
+        "ipRestrictionRules": {
```

Comment:
Remove changes from this. We cannot add changes to older api versions. Limit your changes to  2022-05-01.
We will add your changes to future api-versions which are not published yet

Reply:
done